packit_service/worker/build/copr_build.py

```diff
@@ -620,6 +620,34 @@ def create_copr_project_if_not_exists(self) -> str:
             for field, (old, new) in ex.fields_to_change.items():
                 table += f"| {field} | {old} | {new} |\n"
 
+            chroots_diff = ""
+            if "chroots" in ex.fields_to_change:
+                old_chroots, new_chroots = ex.fields_to_change["chroots"]
+                chroots_diff = "Diff of chroots:\n```diff\n"
+                extra_chroots = set(old_chroots).difference(new_chroots)
+                missing_chroots = set(new_chroots).difference(old_chroots)
+                for e in extra_chroots:
+                    chroots_diff += f"-{e}\n"
+                for m in missing_chroots:
+                    chroots_diff += f"+{m}\n"
+                chroots_diff += "```\n"
```

Comment:
It looks really nice!

Can you please move this to a separate function/method? This one is getting really long...